controller/adr.rst

```diff
@@ -0,0 +1,187 @@
+.. index::
+single: Action Domain Responder approach
+
+How to implement the ADR pattern
+================================
+
+In Symfony, you're used to implement the MVC pattern and extending the default 'Controller'
+class, since the 3.3 update, Symfony is capable of using natively the ADR approach.
+
+Updating your internal logic
+----------------------------
+
+As you saw from earlier example, you must update the services.yml file in order to
```

Comment:
"As you saw from earlier example" does not mean anything is this context what introducing by:
```rst
You just need to update `the services.yml` file in order to auto configure your actions::
```
(Notice there is no space before the `:: `.